karma.conf.js

```diff
@@ -1,49 +1,7 @@
 'use strict';
 
-var RewirePlugin = require('rewire-webpack');
-var webpackConfig = require('./webpack.config.js');
-
-// Remove the bits from the shared config
-// that we don't want for tests.
-delete webpackConfig.output;
-delete webpackConfig.entry;
-
-// Add this just for testing:
-webpackConfig.plugins = [
-  new RewirePlugin(),
-];
-
+var karmaConfig = require('./karma.config.shared');
```

Comment:
same comment here: ``karma.conf.shared`` might be a bit more consistent (just a nit)